frontend/src/metabase/static-viz/components/XYChart/types.ts

```diff
@@ -48,6 +48,7 @@ export type ChartSettings = {
     value: number;
     label: string;
   };
+  show_values?: boolean;
```

Comment:
This prop could be changed later if BE decided it's best to expose `show_values` somewhere else or in any other way.